common/browser.go

```diff
@@ -144,6 +144,18 @@ func (b *Browser) disposeContext(id cdp.BrowserContextID) error {
 	return nil
 }
 
+// getBrowserContextForPageByID returns the BrowserContext for the given page ID.
+// If the browser context is not found, the default BrowserContext is returned.
+func (b *Browser) getBrowserContextForPageByID(id cdp.BrowserContextID) *BrowserContext {
```

Comment:
This doesn't seem to be related to a page or a page id. Should it be `getBrowserContextByID`?

Reply:
How about `getBrowserContextByIDOrDefault`? The reason is this also returns a default browser context if it cannot find the browser context. I'm open to other naming suggestions.

Reply:
I've changed it to `getDefaultBrowserContextOrByID`, [here](https://github.com/grafana/xk6-browser/pull/848/commits/1a167e79d711901df566f34332c82ba8c0d50c60). WDYT?